core/SCsub

```diff
@@ -109,6 +109,11 @@ thirdparty_minizip_sources = ["ioapi.c", "unzip.c", "zip.c"]
 thirdparty_minizip_sources = [thirdparty_minizip_dir + file for file in thirdparty_minizip_sources]
 env_thirdparty.add_source_files(thirdparty_obj, thirdparty_minizip_sources)
 
+# LZ4 library
+if env["builtin_lz4"]:
+    env_thirdparty.add_source_files(thirdparty_obj, "#thirdparty/lz4/lz4.c")
+    env.Append(CPPDEFINES=["LZ4_ENABLED"])
```

Comment:
Using this constant here is wrong, `builtin_` is for switching between system version of the library and built-in one, not for disabling features. So it should be always enabled (this a tiny lib, so this is probably a way to go) or have a separate flag to enable/disable it.

Reply:
Nice to know, I still included an option to disable it anyway